x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/saml/SamlMetadataCommandTests.java

```diff
@@ -679,6 +718,30 @@ public void testDefaultOptionsWithSigningAndMultipleEncryptionKeys() throws Exce
         }
     }
 
+    public void testWrongKeystorePassword() {
+        assumeFalse("Can't test this when the keystore isn't password protected", passwordProtectedKeystore == false);
```

Comment:
Can we create a different keystore that will always be password protected here? We shouldn't rely on randomization for this test tor un at all.